packages/gatsby-codemods/src/transforms/__testfixtures__/gatsby-plugin-image/extends-warn.output.js

```diff
@@ -8,7 +8,7 @@ const svgAssetsContext = require.context(
   /^\.\/.*\.svg$/
 )
 
-class SVGGatsbyImage extends Image {
+class SVGGatsbyImage extends GatsbyImage {
```

Comment:
From the filename, I'm guessing this will warn, right? This will break of course, because `GatsbyImage` is a function component not a class.